File: cpp/hello.cpp
#include<iostream>

// using namespace std;

int main(){
    std::string name;

    std::cout<<"What is your name?"<<std::endl;
    std::cin >> name;
    std::cout<<"Hello " + name + ",  welcome to the world of C++"<<std::endl;

    return 0;
}

File: c#/hello.cs
using System;


///<summary>Główna Klasa Aplikacji</summary>
public class HelloWorld
{

    public static void Main(string[] args)
    {
        string name;
        Console.WriteLine("What is your name");
        try 
        {
            name = Console.ReadLine();
            Console.WriteLine("Hello " + name + ", welcome to the world of C#");
        }
        catch (Exception e)
        {
            Console.WriteLine(e);
        }
        

        Console.ReadKey();
    }
}